service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/parameters/EncounterTemplateParameters.java

```diff
@@ -16,4 +16,6 @@
     private String code;
     private String displayName;
     private String originalText;
+    private String author;
```

Comment:
author is not used in this PR but it does not matter as soon as #156 will be merged first.